README.md

```diff
@@ -31,6 +31,20 @@ There's an external project [docker-lobsters](https://github.com/utensils/docker
   On MacOS you can [install with brew](https://mariadb.com/kb/en/installing-mariadb-on-macos-using-homebrew/).
   On Windows there's an [installer](https://mariadb.org/download/?t=mariadb&p=mariadb&r=11.5.2&os=Linux&cpu=x86_64&pkg=tar_gz&i=systemd&mirror=starburst_stlouis).
 
+* Start mysql and set the `root` user password
+
+```sh
+# Start the server
+mysql.server start
+# open a mysql console
+mysql
+```
+
+```sql
+FLUSH PRIVILEGES;
```

Comment:
I think the `flush` has to come after the `alter`?

Reply:
Now that I look into it more, the internet disagrees. Some of the answers on [this stackoverflow question](https://stackoverflow.com/questions/36463966) agree with you. I did some experimenting and I didn't actually need this at all. I could change the root pass, see rails fail, change it back to `localdev`, and see rails succeed, all without ever calling `FLUSH PRIVILEGES`.

I'll remove it because it seems not useful, but I'm happy to add it back (at the end) if you'd like.

Reply:
Good find! Yeah, the docs are clear that it's not needed with `GRANT`. Let's leave it out.